src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -531,6 +530,33 @@ protected void prepareSpawnArea(WorldServer world) {
     }
 
     @Override
+    public Optional<World> copyWorld(World originalWorld, String copyName) {
+        try {
+            FileUtils.copyDirectory(getWorldDir(originalWorld.getName()), getWorldDir(copyName));
```

Comment:
Wouldn't the changes since the last auto-save be a larger set of changes on average then the changes between an autosave that you triggered immediately before copying?